pkg/specgen/generate/kube/kube.go

```diff
@@ -393,6 +393,28 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 		}
 	}
 
+	if cpuset, ok := annotations[define.CpusetAnnotation+"/"+opts.Container.Name]; ok {
```

Comment:
Not for this PR, but I wonder if we shouldn't detect annotations referring to nonexistent containers and warn the user... Would help folks rolling YAML by hand